app/pathogen/arbovirus/dashboard/(map)/ArbovirusMap.tsx

```diff
@@ -89,7 +96,17 @@ export function ArbovirusMap(input: ArbovirusMapProps) {
           generatePopupContent={(record) => (
             <ArboStudyPopupContent record={record} />
           )}
-        />
+          dataPoints={state.filteredData}
+          clusterProperties={{
```

Comment:
Since the cluster is not a layer, we need to provide it with information as metadata in the source itself. There is inbuilt functionality in mapbox that will use the below equation to find that number per cluster.